jakarta/rewrite.yml

```diff
@@ -585,7 +585,7 @@ recipeList:
       newValue: 4.0.0-RC1
```

Comment:
Please update to `4.0.0`

Reply:
That should be a different PR I guess?

Reply:
Ah if GraphQL depends on it, then we can do that here as well, I think.

Reply:
I don't know, depends how hard we try to keep dependencies properly aligned :)